Everly Precia Suresh/loops/swap_first_last.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <math.h>
+using namespace std;
+int main(){
+    int n;
+    cout<<"Enter a number: ";
+    cin>>n;
+    int last=n%10;
+    int digits=log10(n);
```

Comment:
It's great you know we can find number of digits using log. But for this assignment you are assumed to solve this problem using loops.